src/main/java/domain/WinningNumbers.java

```diff
@@ -0,0 +1,44 @@
+package domain;
+
+import java.util.List;
+
+public class WinningNumbers implements Numbers {
+
+	private final List<Integer> winningNumbers;
+	private static final int LOTTO_NUMBER = 6;
```

Comment:
상수와 인스턴스의 위치를 구분해주면 가독성이 더 좋을 것 같아요~ 저는 상수를 먼저 선언해놓고 공백라인으로 인스턴스와 구분하는 방식으로 합니다~

```suggestion
	private static final int LOTTO_NUMBER = 6;

	private final List<Integer> winningNumbers;
```

Reply:
제가 클린 코드 10장을 지키지 않았었네요😂

private static이 우선하고, 비공개 인스턴스 변수가 오는 순서가 맞는 것 같습니다.
다른 클래스에서도 모두 적용시키겠습니다.